snitun/utils/ipaddress.py

```diff
@@ -1,9 +1,11 @@
 """Utils for handling IP address."""
 
+from functools import lru_cache
 import ipaddress
 import socket
 
 
+@lru_cache
```

Comment:
The nodes are handling connections from thousands. With the default size of 250, how much benefit is this going to have?

Reply:
Its very effective on the client side. I expect it to be mildly effective on the server since its likely its the same installs being seen over and over that are the most active. Not everyone will be interacting with the system at the same time.